src/pages/home/ReportScreen.js

```diff
@@ -248,111 +251,118 @@ class ReportScreen extends React.Component {
+                            style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
+                            onLayout={(event) => {
+                                const skeletonViewContainerHeight = event.nativeEvent.layout.height;
+
+                                // The height can be 0 if the component unmounts - we are not interested in this value and want to know how much space it
+                                // takes up so we can set the skeleton view container height.
+                                if (skeletonViewContainerHeight === 0) {
+                                    return;
+                                }
+                                reportActionsListViewHeight = skeletonViewContainerHeight;
+                                this.setState({skeletonViewContainerHeight});
+                            }}
+                        >
+                            {this.isReportReadyForDisplay() && !isLoadingInitialReportActions && !isLoading && (
+                                <ReportActionsView
                                     reportActions={this.props.reportActions}
                                     report={this.props.report}
                                     isComposerFullSize={this.props.isComposerFullSize}
-                                    onSubmitComment={this.onSubmitComment}
-                                    policies={this.props.policies}
+                                    parentViewHeight={this.state.skeletonViewContainerHeight}
+                                    policy={policy}
                                 />
-                            </>
-                        )}
-
-                        {!this.isReportReadyForDisplay() && (
-                            <ReportFooter
-                                shouldDisableCompose
-                                isOffline={this.props.network.isOffline}
-                            />
-                        )}
-
-                        <EmojiPicker ref={EmojiPickerAction.emojiPickerRef} />
-                        <PortalHost name={CONST.REPORT.DROP_HOST_NAME} />
-                    </View>
-                </FullPageNotFoundView>
-            </ScreenWrapper>
+                            )}
+
+                            {/* Note: The report should be allowed to mount even if the initial report actions are not loaded. If we prevent rendering the report while they are loading then
+                            we'll unnecessarily unmount the ReportActionsView which will clear the new marker lines initial state. */}
+                            {(!this.isReportReadyForDisplay() || isLoadingInitialReportActions || isLoading) && (
+                                <ReportActionsSkeletonView containerHeight={this.state.skeletonViewContainerHeight} />
+                            )}
+
+                            {this.isReportReadyForDisplay() && (
+                                <>
```

Comment:
I think we can get rid of this fragment, right?

Reply:
Yes, you're right. I didn't notice it because I actually didn't touch this code 😄. The fragment before contains 2 elements, but then changed here https://github.com/Expensify/App/pull/16922